.ci/generate_bear_requirements.py

```diff
@@ -96,7 +106,7 @@ def get_all_requirements(bears):
                requirement not in pip_requirements:
                 pip_requirements.append(requirement)
             elif isinstance(requirement, NpmRequirement) and \
-               requirement not in npm_requirements:
+                requirement not in npm_requirements:
```

Comment:
removing this in subsequent push, waiting for review on the rest and the tests to complete.